### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by zreichert**
- adds in first lesson looping.py
- adds tests for looping.py
- tox configuration
- basic Makefile

### CODE REVIEW

**Comment by zreichert** [comment]
FYI the tests and linter will fail because there is no solution to the problems.  🤷‍♂️

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ - pip install tox-travis
+script: tox
+
+stages:
+  - test
+
+jobs:
+  include:
+  - python: 3.6
+    env: TOXENV=flake8
```
Comment (cowofevil) [inline .travis.yml:16]: Missing newline at EOF. Doesn't matter, but it's important to be pedantic.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	@python -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
+
+develop: ## install necessary packages to setup a dev environment
+	pip install -r requirements.txt
+
+lint: ## check style with flake8
+	flake8 python_practice
+
+test: ## run unit tests quickly with the default Python
+	py.test tests
```
Comment (cowofevil) [inline Makefile:14]: Missing newline at EOF. Doesn't matter, but it's important to be pedantic.

File: python_practice/looping.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+def looping_challenge_three():
+    """Print the following on separate lines
+
+    The numbers 1 - 4 in acceding order
+    The numbers 4 - 1 in descending order
+
+    * Bonus points: accomplish this with only one list that contains only 4 elements
+    """
```
Comment (cowofevil) [inline python_practice/looping.py:31]: FYI: Always use `pass` for boilerplate code.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""The setup script."""
+
+from setuptools import setup
+
+setup(
+    name='python-practice',
+    version='0.0.1',
```
Comment (cowofevil) [inline setup.py:10]: Version string mismatch.

File: python_practice/__init__.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+"""Top-level package for python_practice."""
+
+__author__ = """WorldPolice"""
+__email__ = 'cheftyree@gmail.com'
+__version__ = '0.0.0'
```
Comment (cowofevil) [inline python_practice/__init__.py:7]: Version string mismatch.

File: python_practice/looping.py (hunk 3 above)
Comment (cowofevil) [reply python_practice/looping.py:31]: Also, awesome question. Got me thinking on all the ways you could accomplish this.

**Review by cowofevil — APPROVED** [review]

**Pull request merged**